java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -156,6 +156,23 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> mset(Map<String, String> keyValueMap);
 
+    /**
+     * Sets multiple keys to values if the key does not exist. The operation is atomic, and if one or
+     * more keys already exist, the entire operation fails.
+     *
+     * @apiNote When in cluster mode, all keys in <code>keyValueMap</code> must map to the same hash
+     *     slot.
+     * @see <a href="https://redis.io/commands/msetnx/">redis.io</a> for details.
+     * @param keyValueMap A key-value map consisting of keys and their respective values to set.
+     * @return <code>true</code> if all keys were set. <code>false</code> if no key was set.
+     * @example
+     *     <pre>{@code
+     * Boolean result = client.msetnx(Map.of("key1", "value1", "key2", "value2"}).get();
+     * assertTrue(result);
```

Comment:
```suggestion
     * assert result;
```
`assertTrue`, `assertEquals` and other assert functions are for tests.